dallinger/command_line.py

```diff
@@ -423,8 +435,8 @@ def deploy_sandbox_shared_setup(verbose=True, app=None, web_procs=1):
         "heroku config:set heroku_email_address=" +
         config.get('Heroku Access', 'heroku_email_address'),
 
-        "heroku config:set heroku_password=" +
-        config.get('Heroku Access', 'heroku_password'),
+        'heroku config:set heroku_password="' +
+        config.get('Heroku Access', 'heroku_password') + '"',
```

Comment:
We should escape any double quotes in the password. Well, really we should change how all these commands are run to avoid the possibility of injecting additional arguments.